src/Missings.jl

```diff
@@ -50,6 +51,31 @@ missings(dims...) = fill(missing, dims)
 missings(::Type{T}, dims...) where {T >: Missing} = fill!(Array{T}(dims), missing)
 missings(::Type{T}, dims...) where {T} = fill!(Array{Union{T, Missing}}(dims), missing)
 
+"""
+    allowmissing(x::AbstractArray)
+
+Return an array equal to `x` allowing for [`missing`](@ref) values,
+i.e. with an element type equal to `Union{eltype(x), Missing}`.
+
+When possible, the result will share memory with `x` (as with [`convert`](@ref)).
+
+See also: [`disallowmissing`](@ref)
+"""
+allowmissing(x::AbstractVector{T}) where {T} = convert(AbstractArray{Union{T, Missing}}, x)
```

Comment:
Oops. Didn't catch this until after I merged. I think these are supposed to be `x::AbstractArray` as shown in the docstring. I'll open a PR with a fix